lib/editing/editor_codemirror.dart

```diff
@@ -116,20 +116,39 @@ class CodeMirrorFactory extends EditorFactory {
         );
       }).toList();
 
+      List<HintResult> filterHints = new List.from(hints, growable: true);
+
+      // If we have 2 hint results with the same displaytext
+      // where one is a getter and the other setter,
+      // we merge them as 1 result with classname: "type-property"
+      for (HintResult result in hints) {
+        for (HintResult other in hints) {
+          if (result.displayText  == other.displayText && result.className == "type-getter" && other.className == "type-setter") {
+            filterHints.removeWhere((e) => result.displayText == e.displayText);
+            filterHints.add(new HintResult(
```

Comment:
Actually - thinking about your comment about ordering - I think we'll want to insert the new hint into the same place in the list as the first item we're removing. That'll preserve the order that was sent from the server.